gallery/src/styles/NxList/NxListPage.tsx

```diff
@@ -93,17 +81,20 @@ const NxListPage = () =>
               with <code className="nx-code">.nx-list__subtext</code>
             </td>
           </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-list-actions</code></td>
```

Comment:
Why isn't this `.nx-list__actions`?

Reply:
> Why isn't this `.nx-list__actions`?

Because it isn't an element, it's a container for other elements

Reply:
> Because it isn't an element, it's a container for other elements

I think it's an element; it's a subpart of a BEM block.  That it contains other things doesn't matter I don't think.

In what way is this a different scenario than the action list in the tile header, which is an element (`.nx-tile__actions`)?

Reply:
> > Because it isn't an element, it's a container for other elements
> 
> I think it's an element; it's a subpart of a BEM block. That it contains other things doesn't matter I don't think.
> 
> In what way is this a different scenario than the action list in the tile header, which is an element (`.nx-tile__actions`)?

It is not, though now I think `nx-tile__actions` is wrong.

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/07c17dfad3d1406150baebe1de357268ee91edf8